### ISSUE TITLE
Google Chrome User Agent

### ISSUE THREAD

**Issue by dgadelha** [Bunkerbewohner/tidal-music-linux#1]
The application needs to fake a Google Chrome User-Agent to make the Hi-Fi streaming work.

**Comment by Bunkerbewohner** [comment]
I can't test it right now, but I added some code to fake the user agent in the new branch [feature/chrome-useragent](https://github.com/Bunkerbewohner/tidal-music-linux/tree/feature/chrome-useragent), see https://github.com/Bunkerbewohner/tidal-music-linux/commit/ad0f2138544cc9c7b7a0d3f379860e4e372e07d9.

So you could try that by checking out the branch locally and then starting the app again:

``` bash
git fetch origin feature/chrome-useragent
git checkout feature/chrome-useragent
npm start
```

(Or something along those lines)

**Comment by dgadelha** [comment]
It resulted into this:

```
Uncaught Exception:
TypeError: Cannot read property 'webRequest' of undefined
    at EventEmitter.<anonymous> (/home/douglas/tidal-music-linux/main.js:54:25)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
```

**Comment by dgadelha** [comment]
This issue has been fixed with my Pull Request #2
Yet, TIDAL still says: **HiFi playback is only available in Google Chrome**

TIDAL is checking for this:
`canPlay:function(){return window.chrome&&void 0!==navigator.mimeTypes["application/x-pnacl"]}`
Do you know how to set both variables? `window.chrome` (empty object) and `navigator.mimeTypes["application/x-pnacl"]`(int 1)

**Comment by dgadelha** [comment]
Any updates on progress setting window.chrome and the mimeType?

**Comment by Bunkerbewohner** [comment]
Sorry, I only just saw your comment about the two variables. 

Setting the variables may be enough to fool the UI, but it doesn't change the fact that NaCl/PNaCl is not supported (see [my comments on your PR](https://github.com/Bunkerbewohner/tidal-music-linux/pull/2#issuecomment-191643883)). 

See for yourself. I updated the branch [chrome-useragent](https://github.com/Bunkerbewohner/tidal-music-linux/tree/feature/chrome-useragent). You will be able to select HiFi streaming, but playback won't work.
